manifests/init.pp

```diff
@@ -185,10 +185,16 @@
       path         => $path,
       alias        => "concat_${name}",
       source       => "${fragdir}/${concat_name}",
-      validate_cmd => $validate_cmd,
       backup       => $backup,
     }
 
+    # Only newer versions of puppet 3.x support the validate_cmd parameter
+    if $validate_cmd {
+      File[$name] {
+        validate_cmd => $validate_cmd,
+      }
+    }
```

Comment:
unless i'm entirely mistaken: given that `validate_cmd`'s default is `undef`, we shouldn't really need the `if` here.